Packs/MandiantAdvantageThreatIntelligence/Integrations/ThreatIntelligence/ThreatIntelligence_test.py

```diff
@@ -0,0 +1,17 @@
+import pytest
```

Comment:
Need to add unit tests for the commands
https://xsoar.pan.dev/docs/integrations/unit-testing